tbox

```diff
@@ -24,14 +24,14 @@ function verify() {
 }
 
 function e2e() {
-  go install
   go test -v ./e2e/... -ginkgo.v -ginkgo.failFast
 }
 
 function build() {
   docker rmi localhost:5000/kubevirt-image-service-exporter:canary --force
   go build
   docker build -f Dockerfile -t localhost:5000/kubevirt-image-service-exporter:canary .
+  rm kubevirt-image-service-exporter
```

Comment:
Does we need to delete the binary?

Reply:
The artifact is not needed after `docker build`. I don't think there is particular reason.